README.md

```diff
@@ -28,12 +28,87 @@ npm i && code . && npm run dev
 
 ### The Problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+1. Navbar Styling:
+
+Problem: Navbar items weren't centered, and the mobile menu wasn't working.
+Solution: Updated CSS classes for proper centering and added mobile menu functionality using React.
+
+2. Component Styling:
+
+Problem: Component styling wasn't affecting the components.
+Solution: Corrected CSS class names in the component styles object to apply the styles.
+Text Alignment:
+
+Problem: Text wasn't centering on mobile and tablet views.
+Solution: Added text-align: center to the appropriate CSS classes.
+
+3. Viewport Height Issue:
+
+Problem: When setting the height to 100vh, white gaps appeared on mobile and tablet views.
+Solution: Adjusted CSS to ensure proper scaling and removed unnecessary minimum width.
+
+4. Multiple Components:
+
+Problem: Concern about setting 100vh height for one section affecting other components below.
+Solution: 100vh will indeed affect other components. You may want to use a more flexible approach, like relative or percentage heights, to avoid unintended effects on other components.
+
+5. Design Explanation:
+
+Problem: Required an explanation of the project's design and its inspiration.
+Solution: Provided a detailed explanation of the Serenity Yoga project's design, including its inspiration and mission.
+
+6. Multipage Components and Routing:
+
+Issue: In the project, there was a requirement to create a web app with multiple pages, each represented by different components. The challenge arose when trying to implement navigation between these pages using React Router.
+
+Solution: To address this, it's crucial to ensure that routes are correctly defined in the React Router configuration. Additionally, the use of the Link component is vital for smooth navigation between different pages.
+
+Context: This issue underscores the common challenge of setting up routing in a multi-page React web app. Properly configuring routes and ensuring seamless transitions between components is essential for a user-friendly experience.
```

Comment:
Your readme is really detailed, but it would be great if you made it more personal as I can see you may have used Chat GPT or similar to write it. Some of the English words are (in my opinion) overly formal and rarely used, this may put some readers off?